lib/gollum/public/gollum/livepreview/js/livepreview.js

```diff
@@ -453,18 +453,17 @@ var applyTimeout = function () {
     var height = $( win ).height();
     var heightHalf = height / 2;
 
-    // height minus 50 so the end of document text doesn't flow off the page.
+    // height minus 40 so the end of document text doesn't flow off the page.
     // + 15 for scroll bar
```

Comment:
This seems to be no longer applicable. :) Would be wise to remove the comment to avoid confusion.